src/components/templates/UiQuestion/UiQuestion.vue

```diff
@@ -249,19 +258,15 @@ const props = defineProps({
     default: () => ({}),
   },
 });
-interface DefaultProps {
-  translation: QuestionTranslation;
-  settings: Questionsettings,
-  iconInfoAttrs: {
-    icon: Icon;
-    [key: string]: unknown
-  },
-  notificationFeedbackAttrs: {
-    type: NotificationType;
-    [key: string]: unknown;
-  },
-}
-const defaultProps = computed<DefaultProps>(() => ({
+const buttonActionAttrs = computed(() => {
+  if (typeof props.settings?.issue?.skip === 'boolean') {
```

Comment:
Personally I think this check is overeager when both Vue and TS force Boolean.

Reply:
Yes but it can be `undefined` and for `undefined` `skip` I want to use `notificationFeedbackAttrs.buttonActionAttrs`.
settings.issue.skip takes precedence over buttonActionAttrs.

Reply:
I think we should change how default settings are merged.
```
issue: ...Object.assign(
  {
    action: false,
    feedback: false,
    skip: true,
  },
  ...props.settings?.issue ?? {}
)
```
This will help both with passing single changes and respecting type.

Reply:
Yup but it is not a problem with types 🙈. 
Here I want to know when the user puts into the settings
```
{
  issue: {
    action: true,
    feedback: true,
  }
}
```
and I use `typeof` to detect this scenario. Because the user can provide onClick for `notificationFeedbackAttrs.buttonActionAttrs` and it want to hide `skip` for some scenario.
Then  it put 
```
{
  issue: {
    skip: false,
  }
}
```
and we are hide this button (to hide button in UiNotification we should pass empty object)

Reply:
If user doesn't provide `skip`, it should use our default value, and it should never be other than Boolean. 🤔
I don't think I understand the benefit of treating `undefined` and `false` differently.